docs/using/visualization/generating_widgets.ipynb

```diff
@@ -575,11 +575,16 @@
    ]
```

Comment:
<p>This should not be placed on same level as that of Line Info Widget. Think about it - since we only need this instruction for line info widget it should be nested within it instead. Just remove the heading, it will automatically nest.</p><p> </p><p>Also, correction: "... to produce Line Info Widget. Thus, ..." not SDEC Plot.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Should I do something wrt to this change @jaladh-singhal ?